go/tools/builders/stdliblist.go

```diff
@@ -158,19 +195,22 @@ func stdliblist(args []string) error {
 		return err
 	}
 
+	execRoot, goroot, err := cloneRoot(".", os.Getenv("GOROOT"))
```

Comment:
"execRoot" is a special name in Bazel. It means the "The working directory for all actions" (https://bazel.build/docs/output_directories). Because you copied the symlink tree to to a different place, it's no longer the actual execRoot. Please rename it to something like "newBase" or "cloneBase".  There are several variables/arguments in this file called "execRoot". After this pull request, they are no longer the actual execroot. Please also renaming all of them.